services/user.py

```diff
@@ -0,0 +1,46 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    user = User(username=username, password=password)
+    if email:
+        user.email = email
+    if password:
+        user.set_password(password)
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(pk=user_id)
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no user is found with the given ID. Consider handling this exception or using `get_object_or_404` if this function is used in a view context.